discovery-infra/log_scrap.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
```

Comment:
lets move all this inside test_infra maybe?

Reply:
not sure what you mean